modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/GridDhtTxFinishFuture.java

```diff
@@ -485,6 +486,12 @@ private boolean finish(boolean commit,
             req.writeVersion(tx.writeVersion() != null ? tx.writeVersion() : tx.xidVersion());
 
             try {
+                if (isNull(cctx.discovery().node(n.id()))){
+                    log.error("Unable to send message (node left topology): " + n);
```

Comment:
We don't log (at least not in all cases) ClusterTopologyCheckedException. Why you've decided to all logging here?